pages/mesosphere/dcos/services/elastic/3.2.0-7.9.3/configuration/index.md

```diff
@@ -0,0 +1,389 @@
+#### Make sure that Kibana is up and running
+
+Services usually take a moment to finish being installed and ready to use. You can check if your Kibana service is ready with the following command:
+
+```bash
+dcos marathon app show kibana | jq -r '.tasksHealthy'
+```
+
+If it outputs a `1` it means Kibana is up and running. A `0` means that it is still probably being installed.
+
+Another good indication that Kibana is ready is when the following line appears in the in the `stdout` log for the Kibana task.
+```
+{"type":"log","@timestamp":"2016-12-08T22:37:46Z","tags":["listening","info"],"pid":12263,"message":"Server running at http://0.0.0.0:5601"}
+```
+
+#### Kibana without X-Pack Security enabled
+
+If Kibana was installed without X-Pack Security enabled, you should be able to access it through the default DC/OS UI Service link (`https://<cluster-url>/service/<kibana-service-name>`).
+
+#### Kibana with X-Pack Security enabled
+
+Due to a known [limitation](/mesosphere/dcos/services/elastic/3.2.0-7.9.3/limitations#kibana-service-url-does-not-work-on-dcos-111), if you installed Kibana with X-Pack Security enabled, you will not be able to access it through the default DC/OS 1.11 UI Service link. In this case you must [expose Kibana using EdgeLB](/mesosphere/dcos/services/elastic/3.2.0-7.9.3/how-to-guides#expose-kibana-using-edgelb).
+
+### Configuration Guidelines
+
+- Service name (`service.name`): This must be unique for each instance of the service that is running. The default is `kibana`.
+- Service user (`service.user`): This must be a non-root user that already exists on each agent. The default user is `nobody`.
+- If you have X-Pack Security enabled in {{ model.techName }} (`elasticsearch.xpack_security_enabled: true`), you must also have it enabled in Kibana (`kibana.elasticsearch_xpack_security_enabled: true`).
+- Elasticsearch credentials (`kibana.user` and `kibana.password`): If you have X-Pack Security enabled, Kibana will use these credentials for authorized requests to Elasticsearch. The default user is `kibana`, and the password must be configured through the service options.
+- Elasticsearch URL: This is a required configuration parameter. The default value `http://coordinator.<elastic-service-name>.l4lb.thisdcos.directory:9200` corresponds to the named VIP that exists when the {{ model.techName }} package is launched with its own default configuration.
+
+### Configuring Kibana
+
+You can customize the Kibana installation in a variety of ways by specifying a JSON options file. For example, here is a sample JSON options file that:
+
+- Sets the service name to `another-kibana`
+- Sets the password for Kibana requests to an Elasticsearch cluster configured with authentication
+- Configures Kibana to [communicate with Elasticsearch via TLS](https://www.elastic.co/guide/en/kibana/current/configuring-tls.html)
+- Turns on X-Pack Security, so that Kibana works against an Elasticsearch similarly configured 
+
+`another_kibana.json`
```

Comment:
I think there needs to be an extra line break between code snippets? Even if it's like the name of what the file would be.

```suggestion
`another_kibana.json`

```